### ISSUE TITLE
Resizing Masks

### ISSUE THREAD

**Issue by kmpape** [georgeoshardo/SyMBac#48]
I observed that after resizing, some masks that were not touching before do touch. I think that this happens on the following line:

https://github.com/georgeoshardo/SyMBac/blob/9513c2cb4dced2f9ae64591627185b9baae79e5f/SyMBac/renderer.py#L403

Is that correct? There are functions to resize and prevent touching, e.g., function `resize_mask` from `delta.imgops`.

**Comment by georgeoshardo** [comment]
Yes it's possible that masks could touch after a resize, especially
depending on how the connectivity is defined, and because I assume you're
using binary masks because of DeLTA?. If you want to avoid it with a quick
fix, I recommend generating data with labelled masks (which are allowed to
touch) and then iterating through each mask and doing morphological erosion
on them one by one, then convert to binary, and re-label, with an assertion
that there are the same number of masks as before.
